cs/src/json/JSON.csproj

```diff
@@ -19,16 +26,22 @@
     <Compile Include="protocols\SimpleJsonReader.cs" />
     <Compile Include="protocols\SimpleJsonWriter.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(BuildFramework)' == 'net45-nonportable'">
```

Comment:
Let me play with it. I tried it that way before and saw some oddities related to project references for dependencies but some other fixes may have addressed that now.